server/models/comment.model.js

```diff
@@ -0,0 +1,23 @@
+const mongoose = require('mongoose');
+const Schema = mongoose.Schema;
+
+const commentSchema = new Schema({
+  post: {
+    type: Schema.Types.ObjectId,
+    ref: 'Post',
+    required: true
+  },
+  author: {
+    type: Schema.Types.ObjectId,
+    ref: 'User',
+    required: true
+  },
+  content: {
+    type: String,
+    required: true
+  }
+}, {
+  timestamps: true
+});
+
```

Comment:
comments has like or dislike functionality. we need to store the number.

Reply:
We are keeping things simple for now. Like/dislike will not be included at this point. We're already behind schedule